docker-compose.yml

```diff
@@ -19,6 +19,34 @@ services:
       REDASH_REDIS_URL: "redis://redis:6379/0"
       REDASH_DATABASE_URL: "postgresql://postgres@postgres/postgres"
       REDASH_RATELIMIT_ENABLED: "false"
+  rq_scheduler:
+    build: .
+    command: dev_rq_scheduler
+    volumes:
+      - type: bind
+        source: .
+        target: /app
+    depends_on:
+      - server
+    environment:
+      REDASH_REDIS_URL: "redis://redis:6379/0"
+  rq_worker:
+    build: .
+    command: dev_rq_worker
+    deploy:
+      replicas: 2
```

Comment:
Left this here for discussion - if one would want to allow multiple workers running simultaneously using `docker-compose`, one would have to run `docker-compose --compatibility up`

Reply:
Isn't `deploy` only for Docker Swarm deployments? Have you considered using `scale` option?

Reply:
If I understand correctly, the `scale` option is deprecated. One could use the `--scale` CLI option, and that would have the same consequence as using `--compatibility`. I just saw the `deploy.replicas` option as a better source of documentation for anyone who would want to scale through docker-compose.

Reply:
You're right. They deprecated it in v3 :unamused: I fail to see the logic behind this move, specially when they recommend _not_ using `--compatibility` in production.

Considering we have no intention of using Swarm -- maybe we should switch to v2?

Reply:
We have at least one thing we need from v3.2 - bind volumes (for proper code reloading), so v2 is not an option. I guess it'll be best to recommend the `--scale` option outside this file. Where do you think would be the best place to do so?

Reply:
Do you remember what's the difference between how we define the volume for `server` and `worker`? I know you added it when working on auto restart, but from looking at the docs I fail to understand the difference.

If it is required, we can consider downgrading the production compose file to v2 as there the scale configuration is more important.

Reply:
`server` is using the default `volume` volume type, because Flask's auto reloading works fine with it. For `worker` (or anything that we use `watchmedo` for) we need a `bind` volume type for FSEvents to play nicely with macOS.

We could definitely use v2 for production compose files though.

Reply:
Ok, as this is all interim -- let's move on and revisit before migrating everything to RQ.

Reply:
OK, for now I've removed the `deploy.replicas` key.